libcloud/test/backup/test_dimensiondata.py

```diff
@@ -35,19 +34,13 @@ def setUp(self):
         self.driver = DimensionData(*DIMENSIONDATA_PARAMS)
 
     def test_invalid_region(self):
-        try:
+        with self.assertRaises(ValueError):
             self.driver = DimensionData(*DIMENSIONDATA_PARAMS, region='blah')
-        except ValueError:
-            pass
 
     def test_invalid_creds(self):
         DimensionDataMockHttp.type = 'UNAUTHORIZED'
-        try:
+        with self.assertRaises(InvalidCredsError):
```

Comment:
I'm fine with using a decorator, but another way of using this method is by passing callable + arguments to it and this method will call your function.

For example:

```python
self.assertRaises(ValueError, DimensionData, *DIMENSIONDATA_PARAMS, region='blah')
self.assertRaises(ValueError, self.driver.list_targets)
```